src/webgl/p5.Shader.js

```diff
@@ -937,7 +937,7 @@ function shader(p5, fn){
       const viewMatrix = this._renderer.states.uViewMatrix;
       const projectionMatrix = this._renderer.states.uPMatrix;
       const modelViewMatrix = (modelMatrix.copy()).mult(viewMatrix);
-      this._renderer.states.uMVMatrix = modelViewMatrix;
+      this._renderer.states.uMVMatrix = this._renderer.states.calculateCombinedMatrix();
```

Comment:
I think this method needs to be on the renderer and not the states:
```suggestion
      this._renderer.states.uMVMatrix = this._renderer.calculateCombinedMatrix();
```